test/Runtime/POSIX/GenRandomBout.c

```diff
@@ -0,0 +1,73 @@
+// RUN: rm -rf file.bout
+// RUN: %gen-random-bout 100 -sym-files 1 10 -sym-arg 4 -sym-files 2 20 -sym-arg 5 -sym-stdin 8 -sym-stdout -sym-arg 6 -sym-args 1 4 5
```

Comment:
Nice test!  But please remove the first occurrence of `sym-files`, I'd like to move to enforcing a single occurrence, as it's a bit confusing now